packages/react-resizable-panels/package.json

```diff
@@ -20,12 +20,18 @@
     ".": {
       "types": "./dist/react-resizable-panels.js",
       "development": {
+        "edge-light": "./dist/react-resizable-panels.development.edge-light.js",
+        "worker": "./dist/react-resizable-panels.development.edge-light.js",
+        "workerd": "./dist/react-resizable-panels.development.edge-light.js",
```

Comment:
Notice those are the same as the one for `node` - the bundle itself is reused. Those keys here are just listed in order of preference (remember those keys here are **ordered** - despite being just object keys)